.github/workflows/ci.yml

```diff
@@ -19,6 +19,11 @@ jobs:
       with:
         go-version: 1.17
 
+    - name: Install package
+      run: |
+        sudo apt-get -y update
+        sudo apt-get -y install genisoimage xorriso
```

Comment:
maybe worth adding `xorriso` or `genisoimage` it to https://github.com/project-flotta/osbuild-operator#requirements ?

Reply:
That will be executed inside the container, so not really needed. No?

Reply:
this yes, but `make build-iso` is run locally.